src/js/components/Distribution/Distribution.js

```diff
@@ -33,6 +33,14 @@ const Distribution = ({
     );
   }
   if (values.length > 1) {
+    for (let i = 0; i < values.length; i += 1) {
```

Comment:
Hmm... We don't usually perform this level of property validation. Instead, we typically try to at least not crash the browser by handling the scenario where there is no value property set. For example:
`const total = values.filter(v => v.hasOwnProperty('value')).reduce(reducer, 0);`

Reply:
Ok I changed the code so instead of throwing and error, it handles when there is no value in values. I am wondering what the default behavior should be for this. Right now if value isn't set in values the first item in values will still be displayed. I am wondering if instead none of the items should be displayed if value isn't set. Also if this happens should there be a warning in the console? Based on the conversation [here](https://grommet.slack.com/archives/C09QQEG69/p1614678325052000?thread_ts=1614624627.045600&cid=C09QQEG69) it may be confusing to users if there isn't something to signify that value needs to be added to values.

Reply:
> I am wondering if instead none of the items should be displayed if value isn't set.

Great idea.

I'm still reluctant to add a console message when no values are provided. We don't do that for DataTable when no `data` is provided or Meter when no `values` are provided.

Reply:
Ok makes sense, I didn't add a console message and I changed distribution so that if value is not set then the items won't be displayed